site/en/guides/thread-primer/node-roles-and-types.md

```diff
@@ -151,9 +151,200 @@ What you learned:
+    <div correct>
+      <div>Minimal Thread Device (MTD).</div>
+      <div>Correct.</div>
+    </div>
+    <div correct>
+      <div>Full Thread Device (FTD).</div>
+      <div>Correct.</div>
+    </div>
+    <div>
+      <div>Miniscule Thread Device (MTD).</div>
+      <div>Incorrect.</div>
+    </div>
+    <div>
+      <div>Sleepy End Device (SED).</div>
+      <div>Incorrect.</div>
+    </div>
+  </devsite-multiple-choice>
+</div>
+
+
+<div>
+  <devsite-multiple-choice>
+    <div>Which of the following statements about Routers are incorrect?</div>
+    <div correct>
+      <div>A Router can disable its transceiver to reduce power.</div>
+      <div>Devices that are functioning as Routers do not disable their
+        transceiver. (If they did, they'd be unable to function properly as a
+        Router.)</div>
+    </div>
+    <div>
+      <div>A Router forwards packets for network devices.</div>
+      <div>Correct.</div>
+    </div>
+    <div>
+      <div>A Router keeps its transceiver enabled at all times.</div>
+      <div>Correct. In order to function properly as a Router, a device must
+        keep its transceiver online at all times.</div>
+    </div>
+    <div>
+      <div>A Router provides secure commissioning services for devices trying 
+      to join the network.</div>
+      <div>Correct. This is an important function of a Router.</div>
+    </div>
+  </devsite-multiple-choice>
+</div>
+
+<div>
+  <devsite-multiple-choice>
+    <div>When can a device upgrade itself to a Router?</div>
+    <div correct>
+      <div>When it is a REED and it is the only node in reach of a new End
+        Device seeking to join the Thread network.</div>
+      <div>That's right. Under these circumstances, a REED can promote itself
+        to a Router.</div>
+    </div>
+    <div>
+      <div>When it is an End Device seeking to join the Thread network.</div>
+      <div>Incorrect.</div>
+    </div>
+    <div>
+      <div>When it is a REED and the Thread network has merged with a larger
+        network.</div>
+      <div>Incorrect.</div>
+    </div>
+  </devsite-multiple-choice>
+</div>
+
+<div>
+  <devsite-multiple-choice>
+    <div>When can a Router cause itself to stop acting as a Router?</div>
+    <div correct>
+      <div>When it has no children.</div>
+      <div>That's correct. A Router with no children is allowed to revert to
+        an End Device on its own.</div>
+    </div>
+    <div>
+      <div>When a new End Device is seeking to join the
+        Thread network.</div>
+      <div>Wrong. A Router cannot revert to an End Device in this scenario.
+      </div>
+    </div>
+    <div>
+      <div>When another device on the network elects to become a Router.
+      </div>
+      <div>Wrong.
+      </div>
+    </div>
+  </devsite-multiple-choice>
+</div>
+
+<div>
+  <devsite-multiple-choice>
+    <div>Consider a scenario where a Thread network contains two 
+    groups of nodes that have radio connectivity with one another, but not
```

Comment:
...two groups of nodes that each have radio connectivity within a group, but not...